sdcm/utils/operator/multitenant_common.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, db_cluster, loaders, monitors,  # pylint: disable=too-many-ar
         self.test_config.reuse_cluster(False)
 
     def get_str_index(self):
-        return f"{self.get_str_index_prefix}-{self.db_cluster.k8s_cluster.tenants_number}-tenants"
+        return f"{self.get_str_index_prefix}-{self.db_cluster.k8s_clusters[0].tenants_number}-tenants"
```

Comment:
shouldn't each tenant know it's dc_idx ?

Reply:
All tenants are participants of all K8S clusters. The `dc_idx` is part of a `DB node`/`pod`, not it's owner/tenant.